tests/chaos/oadp/test_oadp.py

```diff
@@ -39,3 +40,38 @@ def test_reboot_vm_node_during_backup(
     oadp_backup_in_progress.wait_for_status(
         status=oadp_backup_in_progress.Backup.Status.PARTIALLYFAILED, timeout=TIMEOUT_10MIN
     )
+
+
+@pytest.mark.destructive
+@pytest.mark.chaos
+@pytest.mark.parametrize(
+    "rhel_vm_with_dv_running",
+    [
+        pytest.param(
+            {
+                "vm_name": "vm-node-drain-12020",
+                "rhel_image": RHEL_LATEST["image_name"],
+            },
+            marks=pytest.mark.polarion("CNV-12020"),
+        ),
+    ],
+    indirect=True,
+)
+def test_drain_vm_node_during_backup(
+    oadp_backup_in_progress,
+    drain_vm_source_node,
+):
+    """
+    Drain the worker node where the VM is located during OADP backup using DataMover.
+    Validate that backup eventually Completed.
+    """
+    jira_id = "CNV-67519"
+    if is_jira_open(jira_id):
+        oadp_backup_in_progress.wait_for_status(
+            status=oadp_backup_in_progress.Backup.Status.PARTIALLYFAILED, timeout=TIMEOUT_40MIN
```

Comment:
40 minutes? sounds too long for the expected failure

Reply:
From my repeated observations, it indeed takes this long to reliably reach the PartiallyFailed status. However, after reviewing the[ OADP timeout documentation](https://docs.okd.io/latest/backup_and_restore/application_backup_and_restore/troubleshooting/oadp-timeouts.html) and checking the configuration of DPA and Backup in my test environment, I still cannot determine which specific timeout is controlling it. Let me check with the OADP team.